CommonTools/UtilAlgos/interface/EventSelectorAdapter.h

```diff
@@ -30,6 +32,12 @@ class EventSelectorAdapter : public edm::global::EDFilter<>
   // destructor
   ~EventSelectorAdapter() override {}
 
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    T::fillPSetDescription(desc);
+    descriptions.add("@module_type", desc);
```

Comment:
if I understand correctly, when run for a module of type "MyModule" this will result in a `MyModule_cfi.py` file being generated, with the definition of `MyModule = cms.EDProducer("MyModule", ...)`.  
According to some guidelines we used years back, and the more common use case, this should rather go into a `myModule_cfi.py` with a definition like `myModule = cms.EDProducer("MyModule", ...)`.  

Would you consider making the first letter lowercase ?